web-portal/components/SubmissionForm.vue

```diff
@@ -332,7 +332,7 @@
       <!-- CONFIRM EVENT DELETION -->
       <v-dialog v-model="dialog" persistent max-width="300">
         <v-card>
-          <v-card-title class="headline">U shure you wanna delete the event?</v-card-title>
+          <v-card-title class="headline">U sure you wanna delete the event?</v-card-title>
```

Comment:
As many times as I've looked at this file, I can't believe I've never noticed this typo.